TheJumpCloud/jumpcloud-ADMU
# Sa 3327 support for local user account

## Issues
* [SA-3327](https://jumpcloud.atlassian.net/browse/SA-3327) - SA-3327 support for local user account

## What does this solve?
This change fixes bug and allows admins to migrate a JC user with a Local User Account
## Is there anything particularly tricky?
N/A
## How should this be tested?
1. Form: 

- Edit/Create a JC user with a local user account
- Migrate this user with AutoBind set, this should prompt that the user have systemUsername set.
- Press "no" this should take you back to the form then migrate and press "yes"
- Validate that the user has been successfully migrated and should have the system/computer account name = JC SystemUsername

2. CMD:
- Run the test using with a user that has local user account set and autobind
- Validate that the user has been successfully migrated and should have the system/computer account name = JC SystemUsername

3. Run migration with a JC user that have no local user account and autobind = $true
- Validate it is working properly
5. Run migration with a JC user that have no local user account and autobind = $false
- Validate it is working properly


## Screenshots
<img width="1002" alt="image" src="https://github.com/TheJumpCloud/jumpcloud-ADMU/assets/97972790/cfcd3d50-498e-4780-a38e-8f014b57b348">

After migration:
<img width="953" alt="image" src="https://github.com/TheJumpCloud/jumpcloud-ADMU/assets/97972790/4875f67e-7087-43c6-a8a6-8b723d337c8f">




[SA-3327]: https://jumpcloud.atlassian.net/browse/SA-3327?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
